WcaOnRails/env_vars.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+EnvVars = Env::Vars.new(
+  env: {
+    "DISABLE_BULLET" => false,
+    "WCA_LIVE_SITE" => false,
+    "GOOGLE_MAPS_API_KEY" => 'AIzaSyDYBIU04Tv_j914utSX9OJhJDxi7eiZ84w',
+    "GITHUB_CREATE_PR_ACCESS_TOKEN" => '',
+    "STRIPE_API_KEY" => 'sk_test_CY2eQJchZKUrPGQtJ3Z60ycA',
+    "STRIPE_PUBLISHABLE_KEY" => 'pk_test_N0KdZIOedIrP8C4bD5XLUxOY',
+    "STRIPE_CLIENT_ID" => 'ca_A2YDwmyOll0aORNYiOA41dzEWn4xIDS2',
+    "OTP_ENCRYPTION_KEY" => 'abcdefghijklmnopqrstuvwxyz1234567890',
+    "DISCOURSE_SECRET" => 'myawesomesharedsecret',
+    "DISCOURSE_URL" => 'https://forum.worldcubeassociation.org',
+  },
+) do
+  if Rails.env.production?
```

Comment:
Why are these 4 values optional?

Reply:
They should be mandatory, didn't make the changes yet